File: Java-CodeServer/src/main/resources/js/prompto/type/CodeType.js
var NativeType = require("./NativeType").NativeType;

function CodeType()  {
	NativeType.call(this, "Code");
	return this;
}

CodeType.prototype = Object.create(NativeType.prototype);
CodeType.prototype.constructor = CodeType;

CodeType.instance = new CodeType();

/*
@Override
public Class<?> toJavaClass() {
	return null;
}
*/

exports.CodeType = CodeType;

File: Core/src/main/java/prompto/type/CodeType.java
package prompto.type;

import java.lang.reflect.Type;

import prompto.runtime.Context;
import prompto.store.Family;



public class CodeType extends NativeType {

	static CodeType instance = new CodeType();
	
	public static CodeType instance() {
		return instance;
	}
	
	private CodeType() {
		super(Family.CODE);
	}
	
	@Override
	public Type toJavaType(Context context) {
		return null;
	}
}
